test/FakeDataConsumerDAQModule.cpp

```diff
@@ -28,6 +28,9 @@ namespace dunedaq::appfwk {
 FakeDataConsumerDAQModule::FakeDataConsumerDAQModule(const std::string& name)
   : DAQModule(name)
   , thread_(std::bind(&FakeDataConsumerDAQModule::do_work, this, std::placeholders::_1))
+  , nIntsPerVector_(999)
+  , starting_int_(-999)
+  , ending_int_(-999)
   , queueTimeout_(100)
   , inputQueue_(nullptr)
```

Comment:
Should we move initializers for basic-types to the header, using brace syntax?

Reply:
After Slack discussion, agreed that parameters which are only ever meant to be settable via JSON should be set in the header.